sdk/src/types/block/payload/signed_transaction/transaction.rs

```diff
@@ -111,18 +98,28 @@ impl RegularTransactionEssenceBuilder {
         self
     }
 
-    /// Adds a payload to a [`RegularTransactionEssenceBuilder`].
+    /// Adds a payload to a [`TransactionBuilder`].
```

Comment:
tbh this sounds like there could be several, so `Sets ...`?

Reply:
Several what ?
Also you are reviewing an older commit, this has been changed already.

Reply:
fine then :+1: